### PR TITLE
gnome_mplayer: 1.0.4 → 1.0.9

### PR DESCRIPTION
**Pull request by jtojnar**
###### Motivation for this change
Update and change to gtk3.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gnome_mplayer

File: pkgs/development/libraries/gmtk/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      mplayer = "${mplayer}/bin/mplayer";
+    })
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Common functions for gnome-mplayer and gecko-mediaplayer";
+    homepage = https://sites.google.com/site/kdekorte2/gnomemplayer;
+    license = licenses.gpl2;
+    maintainers = gnome_mplayer.meta.maintainers;
+    platforms = platforms.unix;
```
Comment (Mic92) [inline pkgs/development/libraries/gmtk/default.nix]: Either fix support on darwin or exclude this platform, I would say.

Comment (jtojnar) [reply pkgs/development/libraries/gmtk/default.nix]: Need a more complete log

Comment (Mic92) [reply pkgs/development/libraries/gmtk/default.nix]: this is what @grahamc said: https://github.com/grahamc/ofborg/issues/7#issuecomment-349151599

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gnome_mplayer

**Pull request merged**